packages/framework/src/Model/Localization/Localization.php

```diff
@@ -72,15 +88,22 @@ public function getAdminLocale()
     public function getLocalesOfAllDomains()
     {
         if ($this->allLocales === null) {
-            $this->allLocales = [];
-            foreach ($this->domain->getAll() as $domainConfig) {
-                $this->allLocales[$domainConfig->getLocale()] = $domainConfig->getLocale();
-            }
+            $this->loadAllLocalesFromAllDomains();
         }
 
         return $this->allLocales;
     }
 
+    private function loadAllLocalesFromAllDomains(): void
```

Comment:
should be protected to avoid possible merge problems with #788

Reply:
Method removed while solving the previous comment.